SpatialGDK/Documentation/toc.md

```diff
@@ -28,7 +28,9 @@
     - [Directory structure]({{urlRoot}}/content/directory-structure)
     - [Gameplay Ability System]({{urlRoot}}/content/ability-system)
     - [Non-Unreal server-worker types]({{urlRoot}}/content/non-unreal-server-worker-types)
-- <h3>Troubleshooting</h3>
+- <h3>Workflow</h3>
+    - [Local Development Workflow]({{urlRoot}}/content/local-dev-workflow)
+    - [Cloud Development Workflow]({{urlRoot}}/content/cloud-dev-workflow)
```

Comment:
```suggestion
    - [Cloud development workflow]({{urlRoot}}/content/cloud-dev-workflow)
```

Reply:
I was confused between "development" and "deployment" first time round.

(Defo needs lower case initial letters)